$OpenBSD: patch-tests_x509_test_x509_ext_py,v 1.1 2022/01/14 10:59:10 tb Exp $

Skip test that checks that SCT is unsupported which is no longer true
in LibreSSL 3.5.

Index: tests/x509/test_x509_ext.py
--- tests/x509/test_x509_ext.py.orig
+++ tests/x509/test_x509_ext.py
@@ -5608,7 +5608,7 @@ class TestPrecertificateSignedCertificateTimestampsExt
         assert list(ext) == [sct]
 
     @pytest.mark.supported(
-        only_if=lambda backend: backend._lib.CRYPTOGRAPHY_IS_LIBRESSL,
+        only_if=lambda backend: backend._lib.CRYPTOGRAPHY_IS_LIBRESSL and not backend._lib.Cryptography_HAS_SCT,
         skip_message="Requires LibreSSL",
     )
     def test_skips_scts_if_unsupported(self, backend):
